### ISSUE TITLE
Problem of Keras implementation

### ISSUE THREAD

**Issue by Shiweiliuiiiiiii** [wenwei202/iss-rnns#10]
Hi,
Have you ever tried to build dense stacked LSTM on Keras?
I am trying to reproduce the dense big two-stacked LSTM on PTB dataset on keras. With learning rate = 1, the perplexity can reach 220, however, neither using decay 0.1 as your code or the original decay 1/1.15 as the original code can not decrease the perplexity any more. 
Could you please provide any tips for this problem? Thanks.
